src/server/scripts/Spells/spell_priest.cpp

```diff
@@ -1253,28 +1253,73 @@ class spell_pri_prayer_of_mending_jump : public SpellScriptLoader
         {
             PrepareSpellScript(spell_pri_prayer_of_mending_jump_SpellScript);
 
+            enum class TargetPriority : uint8
+            {
+                InjuredPlayers,
+                InjuredPets,
+                FullHealthPlayers,
```

Comment:
where FullHealthPets? :)

Reply:
If there woudl be, it would be last.
Can it be though? fully healthed pets without any fully healthed players or injured players?

Reply:
on second thought: it could be if the pet belongs to the person the aura was on, since he is removed regardless of his health. Will add it.